tutorials/tagger/default-tutorial.html

```diff
@@ -186,7 +186,7 @@ <h2>The Problem</h2>
 {% highlight python %}
     def forward(self,
                 sentence: Dict[str, torch.Tensor],
-                labels: torch.Tensor = None) -> torch.Tensor:
+                labels: torch.Tensor = None) -> Dict[str, torch.Tensor]:
```

Comment:
Oh, I didn't realize there was an html file in here - doesn't this just duplicate what we have in the website repo?

Reply:
yeah, I literally just copy this file into the website repo, maybe it shouldn't be checked in here too? I don't know that it hurts anything

Reply:
A potential issue: manual modifications (like I did to the version on the website to fix this line) could get overwritten.  I guess that's a problem no matter what repo the file is in, though...  But it does seem more error-prone to have two version of the same file in different repos.

Reply:
that file is the output of convert.py, which means that any changes will get overwritten next time the script is updated, let me add a comment to that effect